public/icons/facebook.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Should we use the [Material UI icon](https://material-ui.com/components/material-icons/) for Facebook instead? The same comment applies for the other icons—Twitter, GitHub, Instagram, etc.

Reply:
Makes sense. Switched to Material UI Icons